### PR TITLE
fixed react children issue

### PR DESCRIPTION
**Pull request by cindytung**

### CODE REVIEW

File: src/layouts/index.jsx [hunk 1]
```diff
@@ -7,8 +7,15 @@ import '../styles/base.scss';
 const TemplateWrapper = ({ children }) => (
```
Comment (emesde) [inline src/layouts/index.jsx]: children are automatically passed into props as `this.props.children`

File: src/layouts/index.jsx [hunk 2]
```diff
@@ -7,8 +7,15 @@ import '../styles/base.scss';
...
-      {children}
+    <div
+      style={{
+        margin: '0 auto',
+        maxWidth: 960,
+        padding: '0px 1.0875rem 1.45rem',
+        paddingTop: 0,
+      }}
+    >
+      {children()}
```
Comment (emesde) [inline src/layouts/index.jsx]: `this.props.children`

File: src/layouts/index.jsx [hunk 3]
```diff
@@ -7,8 +7,15 @@ import '../styles/base.scss';
 const TemplateWrapper = ({ children }) => (
   <div>
     <Navbar />
-    <div>
-      {children}
+    <div
+      style={{
+        margin: '0 auto',
+        maxWidth: 960,
+        padding: '0px 1.0875rem 1.45rem',
```
Comment (dav1dn) [inline src/layouts/index.jsx:14]: what is this padding

Comment (cindytung) [reply src/layouts/index.jsx:14]: this is from the init commit lmao

**Pull request closed without merging**